metadata/report/report.go

```diff
@@ -28,8 +28,8 @@ type MetadataReport interface {
 	StoreConsumerMetadata(*identifier.MetadataIdentifier, string) error
 	SaveServiceMetadata(*identifier.ServiceMetadataIdentifier, common.URL) error
 	RemoveServiceMetadata(*identifier.ServiceMetadataIdentifier) error
-	GetExportedURLs(*identifier.ServiceMetadataIdentifier) []string
-	SaveSubscribedData(*identifier.SubscriberMetadataIdentifier, []common.URL) error
-	GetSubscribedURLs(*identifier.SubscriberMetadataIdentifier) []string
-	GetServiceDefinition(*identifier.MetadataIdentifier) string
+	GetExportedURLs(*identifier.ServiceMetadataIdentifier) ([]string, error)
+	SaveSubscribedData(*identifier.SubscriberMetadataIdentifier, string) error
+	GetSubscribedURLs(*identifier.SubscriberMetadataIdentifier) ([]string, error)
+	GetServiceDefinition(*identifier.MetadataIdentifier) (string, error)
```

Comment:
add some comments for this file pls.

Reply:
I shamelessly copy the [comments](https://github.com/apache/dubbo-go/blob/feature/dubbo-2.7.5/metadata/report/nacos/report.go) in original nacos metadata report.